lib/dl_file_uploader_lib/dl_file_uploader_lib_tests/conftest.py

```diff
@@ -44,6 +52,41 @@ def redis_cli(redis_app_settings) -> redis.asyncio.Redis:
     )
 
 
+@pytest_asyncio.fixture(scope="function")
+async def s3_service() -> S3Service:
+    s3_host = get_test_container_hostport("s3-storage", fallback_port=52222).host
+    s3_port = get_test_container_hostport("s3-storage", fallback_port=52222).port
+
+    service = S3Service(
+        access_key_id="accessKey1",
+        secret_access_key="verySecretKey1",
+        endpoint_url=f"http://{s3_host}:{s3_port}",
+        use_virtual_host_addressing=False,
+        tmp_bucket_name="bucket-tmp",
+        persistent_bucket_name="bucket-persistent",
+    )
+
+    await service.initialize()
+
+    client: SyncS3Client = service.get_sync_client()
+
+    try:
+        client.create_bucket(Bucket="bucket-tmp")
+    except client.exceptions.ClientError:
+        pass
+
+    try:
+        client.create_bucket(Bucket="bucket-persistent")
+    except client.exceptions.ClientError:
+        pass
```

Comment:
We have a helper for that https://github.com/datalens-tech/datalens-backend/blob/ab9c1667d71a1905eacd53f516df692999253238/lib/dl_testing/dl_testing/s3_utils.py#L48